core-libs/setup/recipes/b2b/config/default-b2b-occ-config.ts

```diff
@@ -29,8 +23,6 @@ const defaultB2bCartOccEndpoints: CartOccEndpoints = {
 };
 
 const defaultB2bOrderOccEndpoints: OrderOccEndpoints = {
-  scheduleReplenishmentOrder:
```

Comment:
Removed, as we already have this config in the repl checkout (`feature-libs/checkout/scheduled-replenishment/occ/config/default-occ-scheduled-replenishment-config.ts`)